.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,87 @@
+---
+name: 🐛 Bug Report or Question 🤔
+about: Request help/support from the AWS for Fluent Bit team.
+
+---
+<!---
+
+Please remember to first check the documentation and issue history to see if your question/problem has
```

Comment:
might be useful if we provide the links where customer need to check issues first